extra/qcom_gpu_driver/opencl_ioctl.py

```diff
@@ -100,7 +106,7 @@ def ioctl(fd, request, argp):
       mmaped[s.gpuaddr] = mmap.mmap(fd, s.size, offset=s.id*0x1000)
     if name == "IOCTL_KGSL_GPU_COMMAND":
       for i in range(s.numcmds):
-        cmd = get_struct(s.cmdlist+s.cmdsize*i, msm_kgsl.struct_kgsl_command_object)
+        cmd = get_struct(s.cmdlist+ctypes.sizeof(msm_kgsl.struct_kgsl_command_object)*i, msm_kgsl.struct_kgsl_command_object)
```

Comment:
hmm, they are not equal? do we know why?

Reply:
they are not... both on my device and on comma. s.cmdsize shows 64, but the real offset should be the size of the struct = 32. without this, we basically skip to garbage and parse some garbage instead of real command objects

Reply:
okay, that's strange. in the kernel they also use `ptr += sizeof(obj);` and not cmd_size to move it (`kgsl_drawobj_cmd_add_cmdlist`). should `s.objsize` also be changed?

Reply:
yeah, I know... well, just checked it, s.objsize is correct :) 32 just like it is supposed to be on both devices I have access to